test/helpers/wrappers.go

```diff
@@ -77,6 +75,12 @@ func Netperf(endpoint string, perfTest PerfTest) string {
 	return fmt.Sprintf("netperf -l 3 -t %s -H %s", perfTest, endpoint)
 }
 
+// SuperNetperf returns the string representing the super_netperf command to use
+// when testing connectivity between endpoints.
+func SuperNetperf(endpoint string, perfTest PerfTest) string {
+	return fmt.Sprintf("super_netperf 200 -l 30 -t %s -H %s", perfTest, endpoint)
```

Comment:
I have a version of this that takes the number of netperf instances as input. Can we make this
`func SuperNetperf(endpoint string, perfTest PerfTest, instances int) string {...}` Or I can just make it another routine.